tests/python_tests/samples/test_beam_search_causal_lm.py

```diff
@@ -15,7 +15,7 @@ class TestBeamSearchCausalLM:
         "convert_model, sample_args",
         [
             pytest.param("Qwen2-0.5B-Instruct", "你好！", marks=pytest.mark.skipif(sys.platform == "win32", reason="Chinese input failed on Windows")),
-            pytest.param("phi-1_5", "69"),
+            pytest.param("phi-1_5", "69", marks=pytest.mark.skipif(sys.platform == "win32", reason="Subprocess returned non-zero exit status 3221225477 on Windows")),
```

Comment:
please, create a ticket for it.

Reply:
https://jira.devtools.intel.com/browse/CVS-163717